### PR TITLE
Updated utility.inc

### LINKED ISSUE #5543: [PS] Removed unnecessary condition in `views_fetch_fields()`.
## Description of the bug

There is a line (599) in `core\modules\views\includes\utility.inc` contains an unnecessary condition:
```php
if ($string != 'base' && $string != 'base') {
```

## Expected

```php
if ($string != 'base') {
```

### PR DESCRIPTION
**Pull request by aniketh-varma**
Fixes [backdrop/backdrop-issues/#5543](https://github.com/backdrop/backdrop-issues/issues/5543).

Removed unnecessary condition at line (599) in `core\modules\views\includes\utility.inc`

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by klonos** [comment]
@backdrop/core-committers or @jenlampton can we please get one of you to approve running workflows? ^^

Thanks

**Comment by jenlampton** [comment]
I'm not sure what that is, so I'm going to leave this to committers.

**Comment by ghost** [comment]
Thanks! Merged into 1.x and 1.21.x.

**Pull request merged**
